frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { Form, FormTitle, Icon, TitleInput, MessageInput, SubmitInput } from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const [postObject, setPostObject] = useState({});
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setPostObject((values) => ({ ...values, [name]: value }));
+  };
```

Comment:
```suggestion
  const handleChange = (event) => {
    const {name, value} = event.target;
    setPostObject((prevState) => ({ ...prevState, [name]: value }));
  };
```
destrukturyzacja - często się tego używa

plus nazwę prevstate bym inną ustawił